src/server-extension/resolvers/ChannelsResolver/index.ts

```diff
@@ -86,39 +86,37 @@ export class ChannelsResolver {
     @Ctx() ctx: Context
   ): Promise<TopSellingChannelsResult[]> {
     const currentTime = new Date().getTime()
-    const cacheKey = `top-selling-channels-${args?.periodDays}`
-    const cachedData = cache?.get(cacheKey)
-    const dataTTL = cache?.getTtl(cacheKey)
+    const cacheKey = `top-selling-channels-${args?.periodDays}${
+      args?.where && `-${JSON.stringify(args.where)}`
```

Comment:
Other variables also affect the result, like `args.limit` or `info` (resolve tree).
I think caching all possible combinations would just be very inefficient, instead perhaps you can take a look at the query change I proposed below

Reply:
ok, but in the [common usage](https://www.figma.com/file/zQYsZXJo6jmbqcj1kRRmDG/Marketplace?node-id=472-73106&t=AXh6uTQdJ7isMbn0-0) of this query `where` and `limit` mostly will be consts. `periodDays` would be different, but will only have 3 values (7,30, 9999). Assuming that results will be refreshed every minute, would it be proficient to ignore the `info` to save some time on query execution (which I think will be the case for the allTime period)?